apps/src/pythonlab/PythonlabView.tsx

```diff
@@ -72,42 +71,34 @@ const defaultConfig: ConfigType = {
 };
 
 const PythonlabView: React.FunctionComponent = () => {
-  const [currentProject, setCurrentProject] = useState<MultiFileSource>();
```

Comment:
`currentProject` wasn't necessary, it was duplicate state with `source` in redux.